clean_project/search_cli.py

```diff
@@ -1,30 +1,56 @@
 """
 Search a word in a file and return all the occurrence specified the line numbers
 """
+import argparse
 import cmd
-
-import search
+import search_engine
 
 BOLD = '\033[1m'
 END = '\033[0m'
 
 
-class SearchCli(cmd.Cmd):
+class Search(cmd.Cmd):
 
-    def __init__(self):
+    def __init__(self, connect_path):
         cmd.Cmd.__init__(self)
+        self.connect_path = connect_path
 
     def do_search(self, search_word):
         """
         Print all the occurrence of a given word
         :param search_word: Word given to search
         """
         try:
-            results = search.search(search_word)
-            search.print_output(search_word, results)
+            results = search_engine.SearchEngine(self.connect_path).search(search_word)
+            self.print_output(search_word, results)
         except IOError as e:
             print 'File or path not found: %s' % e
 
+    def print_output(self, search_word, results):
```

Comment:
since you use this function for both cli and one-off usage, should probably be put either staticmethod or even better brought outside this class